lib/workload/stateful/ica_event_pipe/stack.ts

```diff
@@ -0,0 +1,54 @@
+import { Construct } from 'constructs';
+import { Stack, StackProps } from 'aws-cdk-lib';
+import { IcaEventPipeConstruct, IcaEventPipeConstructProps } from './construct/ica_event_pipe';
+
+const alarmThreshod: number = 1;
+const queueVizTimeout: number = 30;
+
+/**
+ * IcaEventPipeStackProps
+ */
+export interface IcaEventPipeStackProps {
+  /** The name for stack */
+  name: string;
+  /** The name of the Event Bus to forward events to (used to lookup the Event Bus) */
+  eventBusName: string;
+  /** The name of the SNS Topic to receive DLQ notifications from CloudWatch */
+  slackTopicName: string;
+}
+
+export class IcaEventPipeStack extends Stack {
+  constructor(scope: Construct, id: string, props: StackProps & IcaEventPipeStackProps) {
+    super(scope, id, props);
+    this.createConstruct(this, 'IcaEventPipeConstruct', props);
+  }
+
+  private createConstruct(
+    scope: Construct,
+    id: string,
+    props: StackProps & IcaEventPipeStackProps
+  ) {
+    const constructProps: IcaEventPipeConstructProps = {
+      icaEventPipeName: props.name + 'Pipe',
+      icaQueueName: props.name + 'Queue',
+      icaQueueVizTimeout: queueVizTimeout,
+      eventBusName: props.eventBusName,
+      dlqMessageThreshold: alarmThreshod,
+      slackTopicArn: this.constructTopicArn(props),
+    };
+    return new IcaEventPipeConstruct(scope, id, constructProps);
+  }
+
+  private constructTopicArn(props: StackProps & IcaEventPipeStackProps) {
+    if (!props.env) {
+      throw new Error('No env properties found. Please ensure env.account and evn.region are set.');
+    }
+    if (!props.env.account) {
+      throw new Error('No account');
+    }
+    if (!props.env.region) {
+      throw new Error('No region');
+    }
+    return 'arn:aws:sns:' + props.env.account + ':' + props.env.region + ':' + props.slackTopicName;
```

Comment:
Flo is struggling with bloody, slippery `env` here. 😄 
(All good though, seriously. No action need.)